CSS/Components/modal/modal.css

```diff
@@ -0,0 +1,13 @@
+.modal .card-body {
+    border-bottom: 0.063rem solid #000000;
+}
+
+.modal .card-action {
+    padding: .75rem;
```

Comment:
You can create padding margin utility classes

Reply:
yes I'm planning to add them